DESCRIPTION

```diff
@@ -47,6 +47,10 @@ Authors@R: c(
     person("John", "Kirkpatrick", role = "ctb"),
     person("James", "Black", role = "ctb"),
     person("Leena", "Khatri", role = "ctb"),
+    person("Saddchla", "Mascary", role = "ctb"),
+    person("Zelos", "Zhu", role = "ctb"),
+    person("Jeffrey", "Dickinson", role = "ctb"),
+    person("Ania", "Golab", role = "ctb"),
```

Comment:
these should be `aut`.  I save `ctb` for folks who no longer work on the project